### DISCUSSION TITLE
how to build the gnu emacs with webrender ?

### DISCUSSION THREAD

**Discussion by noor-tg** [emacs-ng/emacs-ng#444] [Q&A]
hey, I want to build the orginial gnu emacs with webrender. how to do it ?

thank you.

**Comment by rawhuul** [comment]
> hey, I want to build the orginial gnu emacs with webrender. how to do it ?
> 
> thank you.
```
$ ./autogen.sh
$ ./configure --with-webrender
$ make -j$(nproc) # proc for number of processors (cores)
```

**Reply by noor-tg** [reply]
this only work with emacs-ng , right ?

**Reply by rawhuul** [reply]
> this only work with emacs-ng , right ?

Yes
